src/Symfony/Component/Form/Extension/Core/Type/BirthdayType.php

```diff
@@ -21,7 +22,14 @@ class BirthdayType extends AbstractType
      */
     public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefault('years', range(date('Y') - 120, date('Y')));
+        $resolver->setDefaults(array(
+            'years' => range(date('Y') - 120, date('Y')),
+            'invalid_message' => function (Options $options, $previousValue) {
+                return isset($options['improved_validation_messages']) && true === $options['improved_validation_messages'] ?
```

Comment:
This condition could be simplified everywhere: 

```diff
- isset($options['improved_validation_messages']) && true === $options['improved_validation_messages']
+ $options['improved_validation_messages'] ?? false
```

Also the ternary should be inlined according to codebase code style

Reply:
Yeah, much cleaner.